app/views/checkout.html

```diff
@@ -0,0 +1,35 @@
+<h2>Checkout</h2>
+<div ng-show="basket.count() > 0">
+  <section id="checkout">
+    <div id="order_items">
+      <table>
+        <tr>
+          <th>Item Name</th>
+          <th>Item Price</th>
+          <th>Item Quantity</th>
+          <th>Subtotal</th>
+          <th></th>
+        </tr>
+        <tr ng-repeat="item in basket.get()">
```

Comment:
@yggie you mean I should have this instead? `<tr ng-repeat="item in basket.get() track by $index">`